### ISSUE TITLE
Samsung TV doesn't work anymore with your blocklist

### ISSUE THREAD

**Issue by abhishekluv** [StevenBlack/hosts#1996]
Hello Steve,

Awesome list. Been using for years.

But, now Samsung TV doesn't work anymore with your blocklist. In pfSense+, I had to ByPass DNSBL for my Samsung TV and only then Youtube and Amazon Prime Video started to work on my TV.

**Comment by StevenBlack** [comment]
Hi Abhishek @abhishekluv thank you for this info.

Are you able to infer which domains seem to be required by Youtube and Amazon Prime on your Samsung TV?

If not, can you tell me approximately when this stopped working?  We could do a diff between various recent versions of this list to isolate which domains may be breaking this.

I'm grateful for any additional information you can provide here.  

It's conceivable that the issue could be locale-specific too.  I don't actually know that, just presenting it as possible at this juncture.

**Comment by abhishekluv** [comment]
Thanks @StevenBlack  for your reply. 

I am based in India. Yes, it may be locale-specific.

Yesterday, I spent 1 hour finding the required domains for YouTube and Prime Video and at the end I just disabled DNSBL for my Samsung TV.

Today, I enabled DNSBL for my TV and my TV is working just fine. I have been using pfSense since 2019 (currently using pfSense+) and I never faced this type of issue. This is the first time when my TV was acting very strangely. Even after unblocking 10+ more URLs in pfBlockerNg YouTube and Prime Video didn't work on my TV. Netflix was working.

Also, I am not novice and quite experienced with blocklists and pfSense. I have used PiHole for 3 years before moving to pfSense and pfBlockerNg.

Anyways, I still keep a watch on my TV and share with you if something comes up.

**Comment by abhishekluv** [comment]
@StevenBlack Your list is awesome like always. Problem was from my side.

I was able to figure it out. It was because of `TLD Allow` feature enabled in `pfBlockerNg`. pfBlockerNg with some blocked TLDs didn't allow unbound DNS to resolve(or sink holed) IPs for my TV for some hours i.e. when the name resolving was happening using `dns.se` or `dns.ae` or something else. I remember my TV YouTube app showing me SSL certificate errors because the app was receiving `10.10.10.1` IP address from pfBlockerNg.

When I enabled DNSBL for my TV today, everything was working properly because the DNS resolving was happening with some DNS from some other country which was allowed in the TLD Allow list. 

I didn't face any issues with `TLD Allow` feature for months. This is the first time something like this happened. Still, I'll be careful with this feature of `pfBlockerNg`. I have switched to Cloudflare and Google DNS so that I can use `TLD Allow` feature.

<img width="764" alt="eeV3oclJjg" src="https://user-images.githubusercontent.com/603975/177048947-b557b23b-8593-4187-8368-bafa0433079c.png">
